lib/v2/curius/links.js

```diff
@@ -0,0 +1,42 @@
+const got = require('@/utils/got');
+const { parseDate } = require('@/utils/parse-date');
+
+module.exports = async (ctx) => {
+    const username = ctx.params.name;
+
+    const name_response = await got({
+        url: `https://curius.app/api/users/${username}`,
+        method: 'get',
+        headers: {
+            Referer: `https://curius.app/${username}`,
+        },
+    });
```

Comment:
got request can be simplified
```suggestion
    const name_response = await got(`https://curius.app/api/users/${username}`, {
        headers: {
            Referer: `https://curius.app/${username}`,
        },
    });
```